app/models/api_docs/service.rb

```diff
@@ -27,6 +27,7 @@ class ApiDocs::Service < ApplicationRecord
   scope :published, -> { where(published: true) }
   scope :accessible, -> { joining { service.outer }.where.has { (service_id == nil) | (service.state != ::Service::DELETE_STATE) } }
   scope :without_service, -> { where(service_id: nil) }
+  scope :swagger2, -> { where('swagger_version >= ?', '2.0') }
```

Comment:
```suggestion
  scope :swagger2, -> { where.has { swagger_version >= '2.0' } }
```
BabySqueel style for `>=` 😄